packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1052,16 +1052,16 @@ abstract class ResidentRunner {
       commandHelp.P.print();
       commandHelp.a.print();
     }
-    if (flutterDevices.any((FlutterDevice d) => d.device.supportsScreenshot)) {
```

Comment:
I think this shouldn't be deleted.

Reply:
It appears twice, just a few lines before this one (sorry, I meant to mention that), but I can put it in another PR if it'll make conflicts.

Reply:
Oh, you're right. I wonder how that happened.